dags/main_summary.py

```diff
@@ -43,6 +45,94 @@
     email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
     dag=dag)
 
+gcloud_docker_image = "google/cloud-sdk:263.0.0-slim"
+main_summary_dataproc_bucket = "gs://moz-fx-data-derived-datasets-parquet-tmp"
+main_ping_bigquery_export_prefix = main_summary_dataproc_bucket + "/export"
+main_ping_bigquery_export_dest = main_ping_bigquery_export_prefix + "/submission_date={{ds}}/document_namespace=telemetry/document_type=main/document_version=4/*.avro"  # noqa
+main_ping_bigquery_export = gke_command(
+    task_id="main_ping_bigquery_extract",
+    command=[
+        "bq",
+        "extract",
+        "--destination_format=AVRO",
+        "moz-fx-data-shared-prod.payload_bytes_decoded.telemetry_telemetry__main_v4${{ds_no_dash}}",
+        main_ping_bigquery_export_dest,
+    ],
+    docker_image=gcloud_docker_image,
+)
+
+main_summary_dataproc = SubDagOperator(
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name="main_summary",
+        dag_name="main_summary_dataproc",
+        default_args=default_args,
+        cluster_name="main-summary-{{ds}}",
+        image_version="1.3",
```

Comment:
I'm assuming this is because you need spark 2.3?

Reply:
yep. the default of `image_version="1.4"` doesn't work because telemetry-batch-view jars are compiled against spark 2.3. Compiling them against spark 2.4 also fixes the issue, but I don't want to block this on upgrading spark for t-b-v.